### PR TITLE
Get single

### PR DESCRIPTION
**Pull request by Sameerthe-Big-O**

### CODE REVIEW

File: config/env.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 const environment = {
   port: process.env.PORT || 2022,
   nodeEnv: process.env.NODE_ENV || "development",
-  mongodbUri: process.env.DB_URI || "mongodb://127.0.0.1:27017/boilerplate",
+  mongodbUri: process.env.DB_URI || "mongodb://127.0.0.1:27017/HRMS",
```
Comment (AbubakarZorrain) [inline config/env.js:4]: Use live mongodb.

File: routes/Hierarchy/controllers.js [hunk 2]
```diff
@@ -14,19 +14,15 @@ const controller = {
...
-      const addResponse = await hierarchy.get(req.params.id);
-      if (addResponse.message === "success") {
-        return httpResponse.SUCCESS(res, addResponse);
-      } else {
-        return httpResponse.UNAUTHORIZED(res, addResponse.data);
-      }
+  get: async (req, res) => {
+    const addResponse = await hierarchy.get(req.params.id);
+    if (addResponse.message === "success") {
+      return httpResponse.CREATED(res, addResponse);
```
Comment (AbubakarZorrain) [inline routes/Hierarchy/controllers.js:20]: Status code for each request?

File: services/hierarchy.js [hunk 3]
```diff
@@ -9,18 +9,16 @@ const hierarchySerive = {
...
-      const savedData = await hierarchyModel.findByIdAndDelete(id);
-      if (savedData) {
-        return { message: "success", data: savedData };
-      }
-    } catch (error) {
-      return { message: "error", data: error.message };
+  get: async (id)=>{
+    try{
+        const data= await hierarchyModel.findById(id);
+        return { message: "success", data };
```
Comment (AbubakarZorrain) [inline services/hierarchy.js:15]: Why return obj with message from service Layer?

File: services/hierarchy.js [hunk 4]
```diff
@@ -31,26 +29,6 @@ const hierarchySerive = {
...
-      if (savedData) {
-        return { message: "success", data: savedData };
-      }
-    } catch (error) {
-      return { message: "error", data: error.message };
-    }
-  },
 };
 
 export default hierarchySerive;
```
Comment (AbubakarZorrain) [inline services/hierarchy.js:34]: Rename  `hierarchySerive` correctly.

**Review by AbubakarZorrain — CHANGES_REQUESTED** [review]

File: routes/Hierarchy/controllers.js (hunk 2 above)
Comment (Sameerthe-Big-O) [reply routes/Hierarchy/controllers.js:20]: sir i'm actually confused what you're to say here like shouldn't i use http response for every type of failure or should i use for every type kindly i'd be better if you'd elaborate

**Review by AbubakarZorrain — CHANGES_REQUESTED** [review]
Resolve Conflicts

**Review by AbubakarZorrain — APPROVED** [review]

**Pull request merged**